src/webgpu/util/texture/texelData.ts

```diff
@@ -173,7 +173,9 @@ const kRepresentationInfo: {
   'rgba32float':            { ...repeatComponents(kRGBA, float32), sRGB: false },
 
   'rgb10a2unorm':           { componentOrder: kRGBA, componentInfo: { R: unorm10, G: unorm10, B: unorm10, A: unorm2 }, sRGB: false },
-  'rg11b10float':           { componentOrder: kRGB, componentInfo: { R: float11, G: float11, B: float10 }, sRGB: false },
+  'rg11b10ufloat':          { componentOrder: kRGB, componentInfo: { R: float11, G: float11, B: float10 }, sRGB: false },
+  // TODO the e5 is shared between all components, figure out how to write it.
+  'rgb9e5ufloat':           { componentOrder: kRGB, componentInfo: { R: float11, G: float11, B: float10 }, sRGB: false },
```

Comment:
I'm going to try to change this to throw exceptions if you try to use this, instead of putting the wrong data.